pkg/sfu/buffer/buffer.go

```diff
@@ -417,7 +431,11 @@ func (b *Buffer) processHeaderExtensions(p *rtp.Packet, arrivalTime int64) {
 	// for bandwidth estimation
 	if b.twcc {
 		if ext := p.GetExtension(b.twccExt); len(ext) > 1 {
-			b.feedbackTWCC(binary.BigEndian.Uint16(ext[0:2]), arrivalTime, p.Marker)
+			sn := binary.BigEndian.Uint16(ext[0:2])
+			marker := p.Marker
+			b.callbackOps <- func() {
```

Comment:
Don't think this will be an issue, but something to watch for. TWCC could be time sensitive. `arrivalTime` is time stamped early. But, the feedback is also sent frequently (every 100 ms) back to the sender. So, this getting stuck in channel should be something we should keep an eye on.